tests/schema/tests.py

```diff
@@ -3,6 +3,8 @@
 import unittest
 from copy import copy
 
+import mock
```

Comment:
`from django.test import mock` -- this uses the package on Python 2 or the built-in version on Python 3.